Lib/test/test_logging.py

```diff
@@ -2813,7 +2813,7 @@ def test_config_9_ok(self):
             logger.info(self.next_message())
             self.assert_log_lines([], stream=output)
             self.apply_config(self.config9a)
-            #Nothing will be output since both handler is still set to WARNING
+            #Nothing will be output since handler is still set to WARNING
```

Comment:
If you're fixing this sentence. I would suggest to also leave a space between `#` and the first word for the sentence.

Reply:
Yes, I agree. I even fixed all the other places in this file. Otherwise, it might look very inconsistent. I'm talking about this last commit https://github.com/python/cpython/commit/53592fc0b8ba08f984de36cd5bd266e13ec0ac27, but it might be overhead and I'm open to revert it.